test/cli/install/registry/bun-install-registry.test.ts

```diff
@@ -3658,94 +3658,103 @@ for (const forceWaiterThread of [false, true]) {
       expect(await exited).toBe(0);
     });
 
-    test.todo("default trusted dependencies should not be used of trustedDependencies is populated", async () => {
-      await writeFile(
-        join(packageDir, "package.json"),
-        JSON.stringify({
-          name: "foo",
-          version: "1.2.3",
-          dependencies: {
-            "uses-what-bin": "1.0.0",
-            // fake electron package because it's in the default trustedDependencies list
-            "electron": "1.0.0",
-          },
-        }),
-      );
+    test.if(breakingChanges_1_0_0)(
```

Comment:
Is this breakingChanges_1_0_0 or breakingChanges_1_1_0?

Reply:
lol yes this should be 1_1_0